src/hook/pauseResume.js

```diff
@@ -196,7 +196,7 @@ eg.module("pauseResume", ["jQuery"], function($) {
 
 			//Clear fx-queue except 1 dummy function
 			//for promise not to be expired when calling stop()
-			$.queue(this, type || "fx", [$.noop]);
+			$.queue(this, type, [$.noop]);
```

Comment:
👍  The variable `type` was intended as a parameter. So it checks validation. But It's not necessary currently as you fixed.